unverified-hostname/unverified-hostname.ts

```
@@ -0,0 +1,42 @@
+      console.log("unverified hostname");
+    },
+  };
+  var request = https.request(options, (response: any) => {
+    let data = "hello";
+    console.log(data);
+  });
+}
+// {/fact}
+
+// {fact rule=unverified-hostname@v1.0 defects=0}
+import https from 'https'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'https'. Consider to delete it or rename it.

'https' is defined here:


(lint/suspicious/noRedeclare)